ql/src/test/results/clientpositive/llap/distinct_groupby.q.out

```diff
@@ -1619,7 +1619,7 @@ STAGE PLANS:
             Execution mode: vectorized, llap
             LLAP IO: all inputs
         Reducer 2 
-            Execution mode: vectorized, llap
+            Execution mode: llap
```

Comment:
this looks like a regression from vectorized to non-vectorized, is it the case?

Reply:
Yes this is a regression, looking into this